### DISCUSSION TITLE
Farming stuck on Synchronization

### DISCUSSION THREAD

**Discussion by FelixOderSo** [guydavis/machinaris#893] [Q&A]
Hi, 
so yea my farming is since I installed Machinaris on Synchronization
and yea, I also gave it a fresh installation didn't fix the problem.
I'm running Machinaris on unraid (newest version) on a Fujitsu PRIMERGY RX300 S8
- 512gb ddr3 ecc ram
- 2x Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz

![image](https://user-images.githubusercontent.com/130495862/231247970-dbd76b16-3d7a-4d4d-be08-780414eb318b.png)

**Comment by guydavis** [comment]
Greetings, yes on a fresh install it can [take days to synchronize](https://github.com/guydavis/machinaris/wiki/Blockchains#blockchain-sync) a Chia blockchain database from network peers.  Alternatively, you can [upgrade](https://github.com/guydavis/machinaris/wiki/Releases#release-streams) to the `machinaris:test` image, which supports a much [faster download](https://github.com/guydavis/machinaris/wiki/Blockchains#blockchain-download) of the entire Chia database.

Either way, please monitor the progress of the sync from the Blockchains page of the Machinaris WebUI.  If successfully syncing, you see the Synced date increasing, getting closer to current time.  You can also monitor the number of network peers, and test your port-forwarding from the [Connections](https://github.com/guydavis/machinaris/wiki/Connections) page.  Hope this helps!

**Comment by FelixOderSo** [comment]
Hi, 
okay, thanks for the answer, so I looked it up in the blockchain tab in the web ui,
Under details, I see the following:

Network: mainnet    Port: 8444   RPC Port: 8555
Node ID: 5237af154cab94c9287e7fea1b2e687f07ed8ffb253b73677638f8d2f70537c3
Genesis Challenge: ccd5bb71183532bff220ba46c268991a3ff07eb358e8255a65c30a2dce0e5fbb
Current Blockchain Status: Syncing 942853/3513025 (2570172 behind).
Peak: Hash: 44a3c7c2b1df5f2a1051170cae14a39606e519851b84326d82cfd18d31d969dc
      Time: Sat Oct 02 2021 13:38:16 PDT                  Height:     942853

Estimated network space: 36.455 EiB
Current difficulty: 3008
Current VDF sub_slot_iters: 133693440

if I see it correct, the synchronization status is this:  942853/3513025 (2570172 behind)

So in unraid I had the option, when I first created the container, to download the blockchain directly instead of syncing it. I set this to “true”. I went all the way through the download process, but yea still on syncing.

**Reply by guydavis** [reply]
Apologies for the trouble.  There is an  issue in Machinaris v0.8.8 (Repository: `ghcr.io/guydavis/machinaris:latest`) related to "blockchain_db_download" setting.

1. Stop the Machinaris container in Unraid Admin UI
2. Completely delete the entire appdata folder off Unraid at `/mnt/user/appdata/machinaris`
3. Edit the Machinaris container, Repository line to `ghcr.io/guydavis/machinaris:test` and ensure "blockchain_db_download" remains set to "true".
4. Start the Machinaris container in the Unraid Admin UI.
5. Monitor the Logs console of the Machinaris container in the Unraid Admin UI.

![image](https://user-images.githubusercontent.com/12936/231784803-fa5f831c-4ce8-43cc-baa5-1751e445294c.png)


This will reset your install and perform the [blockchain download](https://github.com/guydavis/machinaris/wiki/Blockchains#blockchain-download).

**Reply by FelixOderSo** [reply]
Okay so I did everything, the installation was without any issues.
![image](https://user-images.githubusercontent.com/130495862/231824296-3ed252de-d77c-4f93-b262-f5c2542aaf54.png)

**Comment by FelixOderSo** [comment]
Okay it's back online now it says the following under Blockchains:

Network: mainnet    Port: 8444   RPC Port: 8555
Node ID: 60578726ca2613644512a3df1d2d2fa44b76f0d00bca5b464c05a3b16680e0d4
Genesis Challenge: ccd5bb71183532bff220ba46c268991a3ff07eb358e8255a65c30a2dce0e5fbb
Current Blockchain Status: Syncing 36505/3517382 (3480877 behind).
Peak: Hash: 3519e8ecce30f5f25b0db65d589d3139db9286380ff1a1b12ee224aaa587251a
      Time: Fri Mar 26 2021 13:25:59 PDT                  Height:      36505

Estimated network space: 151.988 PiB
Current difficulty: 10
Current VDF sub_slot_iters: 112197632

I guess I'll let it run for a few days until it's synced:)
